### PR TITLE
README: replace alpha annotations to beta

### PR DESCRIPTION
**Pull request by stlaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ data:
 kind: ConfigMap
 metadata:
   annotations:
-    service.alpha.openshift.io/inject-cabundle: "true"
+    service.beta.openshift.io/inject-cabundle: "true"
```
Comment (adambkaplan) [inline README.md]: Why are we api-freezing with `beta`? Can't we drop this and make the annotation `service.openshift.io/inject-cabundle`?

Comment (enj) [reply README.md]: Discussed in https://bugzilla.redhat.com/show_bug.cgi?id=1704883#c1

**Pull request merged**